### DISCUSSION TITLE
Java Binding - I can't build without whisper.dll

### DISCUSSION THREAD

**Discussion by LosokosG** [ggml-org/whisper.cpp#1383] [Bindings]
I cannot build the project in order to get my required .jar file.
Both tests are failing.
Here is the error:
`java.lang.UnsatisfiedLinkError: Unable to load library 'whisper':
The specified module could not be found.

The specified module could not be found.

The specified module could not be found.`

I understand that it's because i don't have the whisper.dll file in the build/generated/resources, but paradoxically i can only get it by building the project.
I tried:
- Re-installing gradle
- Changing JDK version
- Re-downloading
- Using a different PC

If you need, here is the full build scan:
https://scans.gradle.com/s/umbpqzz5tb2hu

**Comment by LosokosG** [comment]
Honestly, if someone could just give me the .jar file, that would be just great! ( if that would work, i have no idea )

**Comment by yuvalkry** [comment]
The DLL in whisper.cpp/releases is not compatible with the Java bindings, the bindings are probably more updated.
